modules/ROOT/pages/introduction/introduction-to-anypoint-connectors.adoc

```diff
@@ -0,0 +1,122 @@
+
+For more information about the HTTP Listener and the Scheduler component, see the xref:http/http-connector.adoc[HTTP Connector User Guide] and xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint (Trigger)].
+
+[[operations]]
+=== Operations
+
+Most connectors have operations that execute API calls or other actions on their associated system, database, or protocol. For example, you can use Anypoint Connector for Workday (Workday Connector) to create a position request in Workday or add a fund to the financial management service. Likewise, you can use Anypoint Connector for VMs (VM Connector) to consume messages from and publish messages to an asynchronous queue.
+
+To see a list of operations for a connector, see the Reference Guide for that connector.
+
+== Access Connectors and Related Assets
+
+Anypoint Exchange provides access to all publicly-available connector assets including connectors, templates, and examples.
+
+=== Connectors in Exchange
+
+You can use Exchange as a starting point for discovering all or a subset of MuleSoft-built connectors.
+
+For information about locating connectors in Exchange, see <<introduction/discover-add-connectors.adoc#Discover and Add Connectors>>.
+
+=== Templates in Exchange
+
+Templates provide starting points for creating Studio projects. Many templates include one or more connector configurations.  For example, the S/4HANA Cloud to Salesforce Partner Migration template helps you migrate partners from an SAP S/4HANA Cloud to a Salesforce instance.
+
+Once you configure a template in Flow Designer or Studio, you can customize it to meet your needs.
+
+For information about locating and using templates, see <<introduction/discover-add-templates.adoc#Discover and Add Templates to Flow Designer or Studio>>.
+
+=== Examples in Exchange
+
+Examples are apps that demonstrate how to handle a use case or solution. Many examples use one or more connectors to accomplish required tasks.
+
+For example, the Authenticating Salesforce using OAuth 2.0 example shows you how to connect to Salesforce using OAuth as the security protocol. It uses the Salesforce Connector to enable OAuth authentication before performing the integration process.
+
+For information about locating and using examples, see <<introduction/discover-add-examples.adoc#Discover and Add Examples to Studio>>.
+
+== Connector Support Categories
+
+Anypoint Connectors have the following categories of support:
+
+* Select connectors are available to anyone. However, you must have an Anypoint Platform subscription to access MuleSoft support for these connectors.
+* Premium connectors are available to licensed users only.
+* Mulesoft Certified connectors are developed by MuleSoft's partners and developer community and subsequently reviewed and certified by MuleSoft. Access and support for these connectors is through MuleSoft partners.
```

Comment:
```suggestion
* Mulesoft Certified connectors are developed by MuleSoft's partners and developer community and subsequently reviewed and certified by MuleSoft. 
Access to and support for these connectors is through MuleSoft partners.
```

Reply:
The first two list items specify "available to." Should the third use the same term?